frontend/packages/console-shared/src/hooks/usePostFormSubmitAction.ts

```diff
@@ -0,0 +1,49 @@
+import { useCallback, useMemo } from 'react';
+import { INCONTEXT_ACTIONS_CONNECTS_TO } from '@console/dev-console/src/const';
+import { K8sResourceCommon } from '@console/internal/module/k8s';
+import { doConnectsToBinding } from '@console/topology/src/utils/connector-utils';
+import { useQueryParams } from './useQueryParams';
+
+/**
+ * Reads an action from the current query params and return a callback to
+ * connect a newly created resource to an existing one in the topology view
```

Comment:
The doc and code indicates relation to topology view through `doConnectsToBinding` function.

Maybe we should rename this file and hook to reflect that, or even move the code under topology package?